content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -15,12 +15,37 @@ menu:
 
 # Upgrade from Gogs
 
-If you are running Gogs which version is below 0.9.99, you can upgrade Gogs to Gitea smoothly. There are some steps to do below:
+If you are running Gogs version 0.9.113 or older, you can upgrade Gogs to Gitea smoothly. There are some steps to do below. Make sure to complete as the gitea user in ~:
```

Comment:
I added recommendation from pull request #57 as I tested this and upgraded without issue to Gitea v1.0.0. Consensus seems to be this is supported.